### ISSUE TITLE
Setup Azure Pipelines for Managed and Native Builds

### ISSUE THREAD

**Issue by simplexidev** [simplexidev/sdevlibs#47]
***This issue replaces #28***
***Depends on #46***
***This issue may be moved to another repository in the future***

Essentially, this is what was left of the last major PR that needs finished, besides the API for libui.

Here is a list of tasks that need done to finish this:

- [x] Setup Managed Builds
  - [x] SDFx
- [ ] Setup Native Builds
  - [ ] `libui-ng`
  - [ ] `miniaudio`
  - [ ] `zstd`

For the native builds, each pipeline will be built on a weekly basis.

**Comment by simplexidev** [comment]
Will open a new issue. This is out-of-date
